Makefile

```diff
@@ -607,20 +607,20 @@ GOLANGCI_LINT = $(shell go env GOPATH)/bin/golangci-lint
 golangci-lint:
 	@[ -f $(GOLANGCI_LINT) ] || { \
 	set -e ;\
-	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b  $(shell go env GOPATH)/bin v1.55.2 ;\
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b  $(shell go env GOPATH)/bin v2.0.2 ;\
 	}
 
 .PHONY: lint
 lint: golangci-lint ## Run golangci-lint and vet linter
-	$(GOLANGCI_LINT) --build-tags "${BUILDTAGS}" run --timeout 10m ./cmd/... ./pkg/... ./kurl_util/...
+	GOOS=linux $(GOLANGCI_LINT) --build-tags "${BUILDTAGS}" run --timeout 10m ./cmd/... ./pkg/... ./kurl_util/...
```

Comment:
can i no longer run this on my mac?

Reply:
This is specifically to allow running the linter on your mac - otherwise it gives errors because there are platform-specific constants that don't exist on mac